README.md

```diff
@@ -1,53 +1,73 @@
 ---
 page_type: sample
 languages:
-- csharp
+- javascript
 products:
-- dotnet
-description: "Add 150 character max description"
-urlFragment: "update-this-to-unique-url-stub"
+- msal
+description: "A simple JavaScript Single-Page Application using @azure/msal-browser"
+urlFragment: "README"

```

Comment:
This needs to be something like `ms-identity-javascript-v2`

Reply:
(it is the url slug used when this sample is published to docs.msft)

Reply:
done